apps/web/src/components/Providers/WebsocketsProvider/index.tsx

```diff
@@ -25,11 +25,12 @@ function useJoinWorkspace({ connection }: { connection: IWebsocketConfig }) {
   const { currentUser } = useSession()
   const { data: workspace } = useCurrentWorkspace()
   return useCallback(() => {
+    if (!workspace?.id) return
```

Comment:
How is possible not to have workspace in private routes?

Reply:
I didn't think of so, should I just assert workspace cannot be undefined?

Reply:
getting the workspace is an async operation so it's normal that it can be null while loading